runtime/src/main/java/org/corfudb/runtime/collections/CorfuQueue.java

```diff
@@ -135,6 +147,24 @@ public boolean equals(Object o) {
             return getEntryId() == that.getEntryId();
         }
 
+        @Override
+        public byte[] generateCorfuHash() {
+            // Note that this hash is used for transaction conflict resolution and shouldn't
+            // generate collisions for different entries
+            // TODO(Maithem) this probably needs the sequence number to
```

Comment:
No you are right - the sequence should not be included here. perhaps you can remove the TODO?